vignettes/laminr.Rmd

```diff
@@ -18,79 +18,51 @@ knitr::opts_chunk$set(
 submit_eval <- laminr:::.get_user_settings()$handle != "testuser1"
 ```
 
-# Introduction
+This vignettes introduce the basic workflow.
 
-This vignettes provides a quick introduction to the **{laminr}** workflow.
-For more details about how **{laminr}** works see `vignette("concepts_features", package = "laminr")`.
```

Comment:
This sentence now comes at the very end of the tutorial so that users don't lose time reading boilerplate.